cloudferrylib/os/compute/nova_compute.py

```diff
@@ -498,6 +498,18 @@ def deploy(self, info, target='instances', **kwargs):
 
         return info
 
+    @staticmethod
+    def _is_flavors_identical(flavor1, flavor1_id, flavor2):
```

Comment:
Yes, I agree. But unfortunately there is no `id` key. I decided that adding `id` key will be more dirty solution than requiring additional argument.